pkg/handlers/generic/lifecycle/ccm/handler.go

```diff
@@ -88,17 +88,39 @@ func (c *CCMHandler) AfterControlPlaneInitialized(
 		log.V(4).Info("Skipping CCM handler.")
 		return
 	}
+
+	clusterConfigVar, _, err := variables.Get[v1alpha1.ClusterConfigSpec](
+		varMap,
+		clusterconfig.MetaVariableName,
+	)
+	if err != nil {
+		log.Error(
+			err,
+			"failed to read clusterConfig variable from cluster definition",
+		)
+		resp.SetStatus(runtimehooksv1.ResponseStatusFailure)
+		resp.SetMessage(
+			fmt.Sprintf("ffailed to read clusterConfig variable from cluster definition: %v",
```

Comment:
```suggestion
			fmt.Sprintf("failed to read clusterConfig variable from cluster definition: %v",
```

Reply:
Fixed in https://github.com/d2iq-labs/cluster-api-runtime-extensions-nutanix/pull/38/commits/cd8dd6d7701a86152c195d0ac150d2ef705ad177